src/Bonsai.ML.Torch/LinearAlgebra/Det.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch.LinearAlgebra
+{
+    /// <summary>
+    /// Computes the determinant of a square matrix.
+    /// </summary>
+    [Combinator]
+    [Description("Computes the determinant of a square matrix.")]
+    [WorkflowElementCategory(ElementCategory.Transform)]
+    public class Det
```

Comment:
```suggestion
    public class Determinant
```

In the spirit of building a consistent terminology, should we call this `Determinant` rather than `Det`? It feels like we are jumping between the terse syntax of TorchSharp and the more verbose canonical Bonsai/C# naming style.